pkg/segment/utils/segconsts.go

```diff
@@ -139,9 +139,10 @@ var VERSION_MBLOCKSUMMARY = []byte{0x01}
 var VERSION_SEGSTATS = []byte{2} // version of the Segment Stats file.
 var VERSION_SEGSTATS_LEGACY = []byte{1}
 
-var VERSION_SEGSTATS_BUF = []byte{3} // version of the single column Seg Stats in a Segment
-var VERSION_SEGSTATS_BUF_LEGACY_1 = []byte{1}
-var VERSION_SEGSTATS_BUF_LEGACY_2 = []byte{2}
+var VERSION_SEGSTATS_BUF = []byte{4} // version of the single column Seg Stats in a Segment
```

Comment:
we should call this V4, because when v5 comes it becomes easier ?

Reply:
addressed in the latest commit.